Description: should be fixed in rustc 1.31 or 1.32, but that requires gdb 8.2
Bug: https://github.com/rust-lang/rust/issues/54614
--- a/src/test/debuginfo/borrowed-c-style-enum.rs
+++ b/src/test/debuginfo/borrowed-c-style-enum.rs
@@ -11,6 +11,7 @@
 
 // compile-flags:-g
 // min-lldb-version: 310
+// ignore-gdb
 
 // === GDB TESTS ===================================================================================
 
--- a/src/test/debuginfo/borrowed-enum.rs
+++ b/src/test/debuginfo/borrowed-enum.rs
@@ -10,7 +10,7 @@
 
 // ignore-tidy-linelength
 // min-lldb-version: 310
-// ignore-gdb-version: 7.11.90 - 7.12.9
+// ignore-gdb
 
 // compile-flags:-g
 
--- a/src/test/debuginfo/c-style-enum-in-composite.rs
+++ b/src/test/debuginfo/c-style-enum-in-composite.rs
@@ -10,6 +10,7 @@
 
 // ignore-tidy-linelength
 // min-lldb-version: 310
+// ignore-gdb
 
 // compile-flags:-g
 
--- a/src/test/debuginfo/gdb-pretty-struct-and-enums-pre-gdb-7-7.rs
+++ b/src/test/debuginfo/gdb-pretty-struct-and-enums-pre-gdb-7-7.rs
@@ -14,6 +14,7 @@
 // ignore-freebsd: gdb package too new
 // ignore-tidy-linelength
 // ignore-lldb
+// ignore-gdb
 // ignore-android: FIXME(#10381)
 // compile-flags:-g
 
--- a/src/test/debuginfo/generic-enum-with-different-disr-sizes.rs
+++ b/src/test/debuginfo/generic-enum-with-different-disr-sizes.rs
@@ -11,6 +11,7 @@
 // ignore-tidy-linelength
 // ignore-lldb: FIXME(#27089)
 // min-lldb-version: 310
+// ignore-gdb
 
 // compile-flags:-g
 
--- a/src/test/debuginfo/generic-struct-style-enum.rs
+++ b/src/test/debuginfo/generic-struct-style-enum.rs
@@ -10,7 +10,7 @@
 
 // ignore-tidy-linelength
 // min-lldb-version: 310
-// ignore-gdb-version: 7.11.90 - 7.12.9
+// ignore-gdb
 
 // compile-flags:-g
 
--- a/src/test/debuginfo/generic-tuple-style-enum.rs
+++ b/src/test/debuginfo/generic-tuple-style-enum.rs
@@ -10,7 +10,7 @@
 
 // ignore-tidy-linelength
 // min-lldb-version: 310
-// ignore-gdb-version: 7.11.90 - 7.12.9
+// ignore-gdb
 
 // compile-flags:-g
 
--- a/src/test/debuginfo/struct-style-enum.rs
+++ b/src/test/debuginfo/struct-style-enum.rs
@@ -10,7 +10,7 @@
 
 // ignore-tidy-linelength
 // min-lldb-version: 310
-// ignore-gdb-version: 7.11.90 - 7.12.9
+// ignore-gdb
 
 // compile-flags:-g
 
--- a/src/test/debuginfo/tuple-style-enum.rs
+++ b/src/test/debuginfo/tuple-style-enum.rs
@@ -10,7 +10,7 @@
 
 // ignore-tidy-linelength
 // min-lldb-version: 310
-// ignore-gdb-version: 7.11.90 - 7.12.9
+// ignore-gdb
 
 // compile-flags:-g
 
--- a/src/test/debuginfo/unique-enum.rs
+++ b/src/test/debuginfo/unique-enum.rs
@@ -10,7 +10,7 @@
 
 // ignore-tidy-linelength
 // min-lldb-version: 310
-// ignore-gdb-version: 7.11.90 - 7.12.9
+// ignore-gdb
 
 // compile-flags:-g
 
